### PR TITLE
Updated readme for issue# 324 and some miscellaneous changes

### PR DESCRIPTION
**Pull request by npanpaliya**
## Checklist before submitting

- [ ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Addressed https://github.com/open-ce/open-ce/issues/324.

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: doc/README.open_ce_build.md [hunk 1]
```diff
@@ -32,13 +32,21 @@ while a similar build for pytorch may look like this:
...
     bazel build ...
 ```
 
+2. By making a global change in user specific `~/.bazelrc` for all packages that use bazel as a build tool. This way one doesn't need to modify every feedstock:
+
+```shell
+    echo "startup --output_user_root=/dev/shm/.cache" >> ~/.bazelrc
+```
+
 Other environment files for other packages can also be found in the `envs`
```
Comment (alexlang74) [inline doc/README.open_ce_build.md:50]: I think this sentence should go before the bazel section

File: doc/README.open_ce_build.md [hunk 2]
```diff
@@ -455,14 +463,14 @@ For example:
...
 
 #### Build packages
-Power10 MMA Optimization is applicable for cpu only builds. One has to use `--ppc_arch=p10` flag in the `open-ce build env` or `open-ce build feedstock` command to build P10 enabled packages. Another important argument which is must to build these packages is `--conda_build_config=open-ce/envs/conda_build_config.yaml,open-ce/envs/conda_build_config_p10.yaml`.[`conda_build_config_p10.yaml`](https://github.com/open-ce/open-ce/blob/main/envs/conda_build_config_p10.yaml) contains Power10 specific settings.
+Power10 MMA Optimization is applicable for cpu only builds. One has to use `--ppc_arch=p10` flag in the `open-ce build env` or `open-ce build feedstock` command to build P10 enabled packages. Another important argument which is must to build these packages is `--conda_build_config=open-ce/envs/conda_build_config.yaml,open-ce/envs/conda_build_config_p10.yaml`.[`conda_build_config_p10.yaml`](https://github.com/open-ce/open-ce/blob/main/envs/conda_build_config_p10.yaml) contains Power10 specific settings.
```
Comment (alexlang74) [inline doc/README.open_ce_build.md:466]: Suggestion:

Power10 MMA Optimization is applicable for cpu only builds. To build Power10 packages, use one of the following options:
- To build an individual environment: `open-ce build env --ppc_arch=p10  --conda_build_config=open-ce/envs/conda_build_config.yaml,open-ce/envs/conda_build_config_p10.yaml <environment yaml>` . Example: `open-ce build env --build_type=cpu --ppc_arch=p10 --conda_build_config=open-ce/envs/conda_build_config.yaml,open-ce/envs/conda_build_config_p10.yaml tensorflow-env.yaml`
- To build an individual feedstock: `open-ce build feedstock --ppc_arch=p10  --conda_build_config=open-ce/envs/conda_build_config.yaml,open-ce/envs/conda_build_config_p10.yaml <feedstock>`
- To build all Power10 enabled packages in one go: `open-ce build env --ppc_arch=p10  opence-p10-env.yaml` :

File: doc/README.open_ce_build.md (hunk 1 above)
Comment (npanpaliya) [reply doc/README.open_ce_build.md]: Done.

File: doc/README.open_ce_build.md (hunk 2 above)
Comment (npanpaliya) [reply doc/README.open_ce_build.md]: Okay. Done.

**Review by alexlang74 — APPROVED** [review]

**Pull request merged**